srcs/cgi/cgiUtil.cpp

```
@@ -0,0 +1,21 @@
+#include "Cgi.hpp"
+
+#include <string>
+#include <set>
+
+bool validCgiFileEnding(const std::string &path)
```

cppcheck

[style] The function 'validCgiFileEnding' is never used.

(unusedFunction)